api/v1/notebook_types.go

```diff
@@ -56,6 +67,7 @@ type NotebookStatus struct {
 
 	// Conditions is the list of conditions that describe the current state of the Notebook.
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+	UploadUrl  string             `json:"uploadUrl,omitempty"`
```

Comment:
this should be UploadURL and uploadURL in json. That's convention that K8s and is used elsewhere in Substratus

Reply:
Good catch, I hadn't spotted the theme